app/routes.php

```diff
@@ -282,6 +283,31 @@ function( Request $request ) use ( $app, $ffFactory ) {
 	}
 );
 
+$app->get(
+	'add-comment',
+	function( Request $request ) use ( $app, $ffFactory ) {
+		$template = $ffFactory->getLayoutTemplate(
+			'Donation_Comment.html.twig'
+		);
+
+		return new Response(
+			$template->render(
+				[
+					'donationId' => (int)$request->query->get( 'donationId', '' ),
+					'updateToken' => $request->query->get( 'updateToken', '' ),
+					'cancelUrl' => $app['url_generator']->generate(
+						'show-donation-confirmation',
+						[
+							'id' => (int)$request->query->get( 'donationId', '' ),
+							'accessToken' => $request->query->get( 'accessToken', '' )
+						]
+					)
+				]
+			)
+		);
+	}
+)->bind( 'AddCommentPage' );
```

Comment:
Added "Consistent route naming" to Planning of 2018.

Reply:
Note that I deviated on purpose to make the distinction between URL and name more clear.